Add SpoolSV documentation (#163)
DavidXanatos/sandboxie-docs: Content/ClosePrintSpooler.md

# Close Print Spooler

_ClosePrintSpooler_ is a sandbox setting that provided nuanced control over how sandboxed applications interact with the print spooler service.

```ini
[DefaultBox]
ClosePrintSpooler=n

> ```

Reviewer: I think it is better to propose the new layout for all documentation settings at once in a separate pull request.

For now, please use the current layout:

https://github.com/sandboxie-plus/sandboxie-docs/blob/225d7162549d37a84b060001e9702e5ccff1a550/Content/AutoDelete.md?plain=1#L4-L10

This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.

> Added as part of [0.5.4 / 5.46.0 version](https://github.com/sandboxie-plus/Sandboxie/blob/f3ab6a73a13a74f894b2d5ceb47ab69ee0581c97/CHANGELOG.md#054--5460---2021-01-06).

Reviewer: ```suggestion
Added as part of 0.5.4 / 5.46.0 version.
```

Author: Will that go on the "improve template" MR?

It's marginally better to have links IMHO :sweat_smile:

Reviewer: In my opinion, putting direct links to the changelog, which in itself is huge, is a bit of a distraction.

## Interaction with [OpenPrintSpooler](OpenPrintSpooler.md)

```ini
[DefaultBox]
ClosePrintSpooler=n
OpenPrintSpooler=n

> ```

Reviewer: I think it is better to propose the new layout for all documentation settings at once in a separate pull request.

For now, please use the current layout:

https://github.com/sandboxie-plus/sandboxie-docs/blob/225d7162549d37a84b060001e9702e5ccff1a550/Content/AutoDelete.md?plain=1#L4-L10

> When both settings are configured as shown above, the sandboxed applications' requests to the print spooler are selectively filtered. This means that certain actions related to the print spooler are permitted ("open") while others are restricted ("closed"). Specifically, this configuration allows for printing operations but restricts activities that would modify printer configurations or the installation/removal of printers on the system.

Reviewer: ```suggestion
When both settings are configured as shown above, requests from sandboxed applications to the print spooler are selectively filtered. This means that certain actions related to the print spooler are permitted ("open") while others are restricted ("closed"). Specifically, this configuration allows for printing operations but restricts activities that would modify printer configurations or the installation/removal of printers on the system.
```

# Close Print Spooler

> _ClosePrintSpooler_ is a sandbox setting that provided nuanced control over how sandboxed applications interact with the print spooler service.

Reviewer: ```suggestion
_ClosePrintSpooler_ is a sandbox setting that provides nuanced control over how sandboxed applications interact with the print spooler service.
```

```
   .
   .
   .
   [DefaultBox]
   ClosePrintSpooler=n
```

> 
> This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
> 
> Added as part of 0.5.4 / 5.46.0 version.

Reviewer: ```suggestion

Added as part of 0.5.4 / 5.46.0 version.

This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
```

Author: I don't know - usually the "added" is "the most useless part" of the documentation
(At least, when it is that wordy).

Does it make sense to emphasize it that much by putting it first?

While the project does not PUSH for updates, it is a "reasonable expectation" that users opt to update?

> ## Interaction with [OpenPrintSpooler](OpenPrintSpooler.md)

Reviewer: ```suggestion
```

Author: I disagree with that change.

I specifically opted in creating a "considerable" visual separator, to indicate that the `ClosePrintSpooler` documentation has finished.

What follows is a "tangent"

```
   .
   .
   .
   [DefaultBox]
   ClosePrintSpooler=n
   OpenPrintSpooler=n
```

> When both settings are configured as shown above, requests from sandboxed applications to the print spooler are selectively filtered. This means that certain actions related to the print spooler are permitted ("open") while others are restricted ("closed"). Specifically, this configuration allows for printing operations but restricts activities that would modify printer configurations or the installation/removal of printers on the system.

Reviewer: ```suggestion
When both settings are configured as shown above, requests from sandboxed applications to the print spooler are selectively filtered. This means that certain actions related to the print spooler are permitted ("open") while others are restricted ("closed"). Specifically, this configuration allows for printing operations but restricts activities that would modify printer configurations or the installation/removal of printers on the system.

See also: [Open Print Spooler](OpenPrintSpooler.md).
```
